backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -631,6 +637,41 @@ public static List<FeedbackMessage> validateGendersOfSexualPartners(ValueOrError
     return errors;
   }
 
+  public static List<FeedbackMessage> validatePositiveHIVRequiredAOEFields(
+      ValueOrError testResult, ValueOrError gendersOfSexualPartners, ValueOrError pregnant) {
+    List<FeedbackMessage> errors = new ArrayList<>();
+    Set<String> positiveTestResultValues = Set.of("positive", "detected");
```

Comment:
would we also need to check for snomed values that are "positive" and "detected" as well here since technically we accept SNOMED values for the test result column

Reply:
Good call out here! Do you know what SNOMED values we should consider valid for "positive" and "detected" results?

It looks like currently `validateTestResult` only checks for whether the value matches an alphabetic regex. If it does, then it also checks that it is among `"positive", "negative", "not detected", "detected", "invalid result"`. If the value is just numbers, then it passes validation automatically.

Reply:
I was checking the livd api: https://prime.cdc.gov/api/metadata/livd
I'm seeing the following on first glance: `10828004` `260373001` but I'm not sure if HIV would be different 🤔